src/dispatch/plugins/dispatch_google/gmail/templates/executive_report.html

```diff
@@ -26,8 +26,6 @@
   <li>{{ next_steps }}</li>
 </ul>
 <p style="font-weight:normal!important;line-height:140%;margin-top:0">
-  A Google document version of the report can be found <a href="{{ weblink }}">here</a>. Please, use the document for questions and feedback.<br><br>
-  Thanks,<br>
-  {{ commander_fullname}}
+  A Google document version of the report can be found <a href="{{ weblink }}">here</a>. Please, use the document for questions and feedback.
```

Comment:
Should we be more generic and drop Google here? There could be different plugins at play (today we just have google).